src/biokbase/narrative/handlers/narrativehandler.py

```diff
@@ -83,7 +83,7 @@ def get(self, path):
                 kill_kernel=False,
                 mathjax_url=self.mathjax_url,
                 google_analytics_id=URLS.google_analytics_id,
-                userName=kbase_env.user,
+                anon_user_id=kbase_env.anon_user_id,
```

Comment:
This line is what this whole chain of PRs is about.

Reply:
One small edit for a programmer, one GA4 leap for programmer kind!